qa-data/invalid-data-encoding/data.csv

```diff
@@ -0,0 +1,4 @@
+id,city
+1,London
+2,Сәлеметсіз бе?
```

Comment:
@anuveyatsu if we do not specify `encoding` in `datapackage.json`, it will be `utf-8` by default. And kazakh alphabet is not supported in `utf-8`